drivers/wifi/nrf700x/rpu_fw_patches.ld

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file
+ * @brief Custom Linker command/script file
+ *
+ * Custom Linker script for the Cortex-M platforms.
+ */
+
+#include <zephyr/linker/sections.h>
+#include <zephyr/devicetree.h>
+
+#include <zephyr/linker/linker-defs.h>
+#include <zephyr/linker/linker-tool.h>
+
+/* TODO: In nRF7002 external flash using SPI isn't supported by nrfjprog yet */
+#if CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP
+
+/*
+ * nRF53 series is shipping a QSPI/SPI external flash mapped at 0x1000_0000 that can
+ * be used for XIP
+ */
+MEMORY
+{
+     EXTFLASH (wx) : ORIGIN = 0x10000000, LENGTH = 0x800000
```

Comment:
When partition manager is used, a part of the external flash may be reserved for some user partitions. For example, Matter samples have two partitions to store DFU images. I think you may want to enable this feature only when the partition manager is enabled and then do sth like:
```
#include <pm_config.h>
#define EXTFLASH_ADDRESS (PM_EXTERNAL_FLASH_ADDRESS + 0x10000000)
#define EXTFLASH_SIZE (PM_EXTERNAL_FLASH_ADDRESS)
...
EXTFLASH (wx) : ORIGIN = EXTFLASH_ADDRESS, LENGTH = EXTFLASH_SIZE
```
to make sure that the patches are put in the unreserved area of the external flash.

Or fall back to the entire area if the partition manager is disabled.

Reply:
Thanks, this is what I was worried about, this clarifies my doubt :).

Reply:
I think so. Just spotted a mistake - `PM_EXTERNAL_FLASH_ADDRESS` => PM_EXTERNAL_FLASH_SIZE in the second last line.